config/application.rb

```diff
@@ -22,6 +22,10 @@ class Application < Rails::Application
   # Do not swallow errors in after_commit/after_rollback callbacks.
   config.active_record.raise_in_transactional_callbacks = true
 
+  # Action Mailer default settings
+  config.action_mailer.default_options = { from: 'noreply@coursemology.org',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.